__tests__/endpoints.test.js

```diff
@@ -45,3 +45,38 @@ describe('GET /api', () => {
         })
     })
 })
+
+describe('GET /api/articles/:article_id', () => {
+    describe('Main functionality', () => {
+        test('when called with an existing id responds with status 200 and the relevant article object', () => {
+            return request(app).get('/api/articles/2')
+                .expect(200)
+                .then((res) => {
+                    expect(typeof res.body.article).toBe('object');
+                    expect(res.body.article).toHaveProperty('author');
+                    expect(res.body.article).toHaveProperty('title');
+                    expect(res.body.article).toHaveProperty('article_id');
+                    expect(res.body.article).toHaveProperty('body');
+                    expect(res.body.article).toHaveProperty('topic');
+                    expect(res.body.article).toHaveProperty('created_at');
+                    expect(res.body.article).toHaveProperty('votes');
+                    expect(res.body.article).toHaveProperty('article_img_url');
+                })
+        })
+    })
+    describe('Error handling', () => {
+        test('when given syntactically correct but non-existent parameter responds with an empty object articles object on res.body', () => {
+            return request(app).get('/api/articles/532939827')
+            .then((res) => {
+                expect(res.body.article).toEqual({});
+            })
```

Comment:
A great test case to have - but your assertion here isn't exactly what we'd be after in a case like this. We'd want to test for:
- A 404 status code on the response
- An error object to be sent back with a message of 'not found'

Instead of just an empty body - we'd want more confirmation of what went wrong which is why we test for an error object and status code